### DISCUSSION TITLE
The focus in a SelectOneMenu element is lost when selecting with the mouse.

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#128] [PrimeFaces] [answered]
In the ShowCase for SelectOneMenu, focus is lost when an entry is selected with the mouse. In version PF 11 the focus is still retained. Is something broken or do I need to adjust something?

**Comment by FlipWarthog** [comment]
That looks like a bug and is present in PF14 as well. Can you please open an issue for it? https://github.com/primefaces/primefaces/issues

Comparing with the PrimeVue equivalent, focus is retained after selection with the mouse: https://primevue.org/dropdown/

**Reply by FlipWarthog** [reply, marked as answer]
Issue created: https://github.com/primefaces/primefaces/issues/10721

**Comment by hgfrey** [comment]
Thank you!
